### PR TITLE
Null on error when converting a property from string to another type.

### PR DESCRIPTION
**Pull request by pstalcup**
Add "validate" method for proxy class types to verify their
string representation matches what is in the property

### CODE REVIEW

File: src/property.ts [hunk 1]
```diff
@@ -43,13 +51,24 @@ type MafiaClasses =
...
   Type: typeof MafiaClass & (new () => T)
 ): ((property: string, default_?: T | null) => T | null) =>
   createPropertyGetter((value) => {
     if (value === "") return null;
-    const v = Type.get<T>(value);
-    return v === Type.get<T>("none") ? null : v;
+    try {
+      const v = Type.get<T>(value);
+      return v === Type.get<T>("none") ? null : v;
+    } catch {
```
Comment (horrible-little-slime) [inline src/property.ts:69]: Will this lead to a Mafia briefly flashing red every time we get an error? I'm fine with that, of course, just curious

**Review by horrible-little-slime — COMMENTED** [review]
This looks great! I am not comfortable approving it myself, but this reads like it does exactly what I want

File: src/property.ts (hunk 1 above)
Comment (gausie) [reply src/property.ts:69]: What exception is thrown here? Can you narrow the catch expression?

**Comment by horrible-little-slime** [comment]
Resolved by #217

**Pull request closed without merging**